src/erlcloud_cloudformation.erl

```diff
@@ -405,7 +404,8 @@ convert_param({Key, Value}) ->
         stack_name -> {"StackName", Value};
         stack_id -> {"StackName", Value};
         logical_resource_id -> {"LogicalResourceId", Value};
-        next_token -> {"NextToken", Value}
+        next_token -> {"NextToken", Value};
+        _ -> {Key, Value}
```

Comment:
do we really need this?
as we may just ask for for all params to be actual `{"", ""}`